### DISCUSSION TITLE
Quicker way for users to quickly rename requests by clicking on them

### DISCUSSION THREAD

**Discussion by denispadua** [Kong/insomnia#6789] [Feedback]
I think it will be a great option if there is a way to block renaming of a request. Most people probably don't rename their request very often, but it's very annoying to click on a request and the text bar appears instead a request.

**Comment by subnetmarco** [comment]
We will change this into a double-click as older Insomnia versions implemented.

**Comment by subnetmarco** [comment]
This has been implemented a long time ago.
